docs/howto/object-storage/s3/presign.md

```diff
@@ -0,0 +1,167 @@
+    `mc` also supports the suffix `m` for minutes, `h` for hours, and `d` for days.
+
+    If you do not set `--expire`, `mc` defaults to creating a pre-signed URL that is valid for 7 days.
+=== "s3cmd"
+    > For pre-signed URLs generated with `s3cmd` to work correctly in {{brand}}, you must use an `s3cmd` version later than 2.0.1, and set `signurl_use_https = True` in your configuration file.
+
+    ```bash
+    s3cmd -c ~/.s3cfg-<region> signurl s3://<bucket-name>/<object-name> +<seconds>
+    ```
+    Note the `+` prefix on the expiry.
+    `s3cmd` also supports setting an absolute expiry date, which does not use the `+` prefix and must be formatted in [Unix time](https://en.wikipedia.org/wiki/Unix_time).
+
+The command will return the valid pre-signed URL.
+
+
+## Accessing objects with a pre-signed URL
+
+To access an object in a public bucket from a web browser or a generic HTTP/HTTPS client like `curl`, open the URL that the pre-sign command returned:
+
+```console
+curl -f -O https://s3-<region>.{{brand_domain}}:8080/<bucket-name>/<object-name>?AWSAccessKeyId=<access-key>&Signature=<signature>&Expires=<expiry>
+```
+
+As long as the query parameters are correct and the signature has not yet expired, this command will succeed.
+If the query parameters are incorrect or the pre-signed URL is past its expiry date, it will fail with HTTP 403 (Forbidden) instead.
+
+> Pre-signed URLs are valid for HTTP `GET` requests only.
+> Thus, even a valid pre-signed URL will result in HTTP 403 if you force a different HTTP method (such as `HEAD`, by setting `curl -I`).
+
+For example, to retrieve an object named `bar.pdf` in a bucket named `foo` in the {{brand}} Kna1 region via its pre-signed URL, you would run:
+
+```console
+$ curl -o bar.pdf https://s3-kna1.{{brand_domain}}:8080/foo/bar.pdf?AWSAccessKeyId=07576783684248f7b2745e34356c6025&Expires=1673521496&Signature=%2Frm9nLV3moP%2FQz7aGCAnrESXjbk%3D
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100 62703  100 62703    0     0   186k      0 --:--:-- --:--:-- --:--:--  186k
+```
+
+## Setting the download file name on the object
+
+When you use `curl` (and potentially other browsers or HTTP/HTTPS clients) to download an object from a pre-signed URL, the default behavior is to set its filename to the name of the downloaded object *including* the query parameters.
+This can lead to rather unwieldly filenames.
+
+To ensure that an object named `bar.pdf` in a bucket named `foo` is always downloaded and stored with `bar.pdf` as its filename, you can set its `Content-Disposition` header.
+
+### Setting `Content-Disposition` on object creation
+
+=== "aws"
+    ```bash
+    aws --profile <region> \
+      --endpoint-url=https://s3-<region>.{{brand_domain}}:8080 \
+      s3api put-object \
+      --content-disposition 'attachment;filename="bar.pdf" \
```

Comment:
I see the left `'` but not the right `'` :/

Reply:
Oh **nice catch!** I am impressed.